data/src/main/java/com/linkedin/data/schema/PathSpecSet.java

```diff
@@ -0,0 +1,377 @@
+package com.linkedin.data.schema;
```

Comment:
Might as well add copy-right file headers?

Reply:
+1. Add copyright comment at the top of all new source files. See existing files for reference.